app/views/carts/_cart_items.html.erb

```diff
@@ -0,0 +1,80 @@
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>Item</th>
+      <th>Quantity</th>
+      <th>Price</th>
+      <% if view==='cart'%>
+        <th></th>
+      <%end%>
+      </tr>
+  </thead>
+  <tbody>
+    <% items.each do |item|%>
+      <tr>
+        <th> <strong> <%= link_to item[:title],item_path(item[:id])%> </strong></th>
+        <th><%= item[:quantity]%></th>
+
+        <th><%= item[:price]*item[:quantity]%></th>
```

Comment:
Make a method and render the result here.

Reply:
Done